File: src/main/kotlin/com/ampnet/auditornode/TestScript.kt
package com.ampnet.auditornode

// language=js
val JS_CODE = """
        try {
            console.log("Hello from JS!");
            var http = Java.type('com.ampnet.auditornode.scriptapi.Http');
            console.log(Object.getOwnPropertyNames(http));
            
            var response = http.request("GET", "https://bb.dom-l.at/");
            console.log("Response in JS:");
            console.log(response);
        } catch (e) {
            console.log("Error: " + e);
        }
    """.trimIndent()

File: src/integTest/resources/test-script.js
function audit() {
    try {
        let response = http.get("https://bb.dom-l.at/");
        console.log("Response body:");
        console.log(response);
        return AuditResult.of(true);
    } catch (e) {
        console.log("Error: " + e);
        return AuditResult.of(false);
    }
}
